experimental/traffic-portal/src/app/api/profile.service.ts

```diff
@@ -103,4 +103,13 @@ export class ProfileService extends APIService {
 		return this.delete<ResponseProfile>(`profiles/${id}`).toPromise();
 	}
 
+	/**
+	 * Import profile
+	 *
+	 * @param importJSON JSON object for import.
+	 * @returns profile response for imported object.
+	 */
+	public async importProfile(importJSON: ProfileImport): Promise<ProfileImportResponse>{
```

Comment:
The ProfileService in the API testing module needs to have this method added as well